src/manifest.json

```diff
@@ -9,42 +9,406 @@
   "children": [
     {
       "type": "group",
-      "id": "redis_stack",
-      "label": "Redis Stack",
+      "id": "ds",
+      "label": "Data structures explained",
+      "args": {
+        "initialIsOpen": false
+      },
+      "children": [
+        {
+          "type": "group",
+          "id": "ds-json",
+          "label": "JSON",
+          "args": {
+            "initialIsOpen": false
+          },
+          "children": [
+            {
+              "type": "internal-link",
+              "id": "ds-json-intro",
+              "label": "Introduction",
+              "summary": "Store, update, and retrieve JSON values in a Redis database, similar to any other Redis data type.",
```

Comment:
@LiorKogan , can you please check the summary for:
- JSON
- Strings
- Hashes
- Lists
- Sets
- Sorted sets
- Time series
- Probabilistic data structures

An example is attached below. For grouped tutorials (JSON, probabilistic, time series) I added the summary only for introductions (the summary can be added only per file, not per folder), I think other file names are self-explaining.

<img width="431" alt="image" src="https://github.com/RedisInsight/Tutorials/assets/99594890/72e8e1fe-93bd-4602-a8c4-f10ddd589cce">

Reply:
- A Redis JSON is a Redis data structure for storing, querying, and manipulating a single JSON document.
- A Redis string, the most basic Redis data type, holds a sequence of bytes.
- A Redis list is a linked list of string values. often used to implement stacks and queues.
- A Redis hash is a collection of fields, where each field is a name-value pair. Both name and value are strings.
- A Redis set is an unordered collection of unique strings.
- A Redis sorted set is a collection of unique strings, ordered by each string's associated score.

Reply:
@ViktarStarastsenka Can you please paste that time series and the probabilistic structures?

Reply:
@LiorKogan 

<img width="434" alt="image" src="https://github.com/RedisInsight/Tutorials/assets/99594890/13ecbd3e-4244-416c-a9f5-83fb9cdda716">

<img width="436" alt="image" src="https://github.com/RedisInsight/Tutorials/assets/99594890/b3576a8d-5f5e-41c4-bb6a-e1c82312148c">